src/test/java/bio/terra/service/dataset/SelfHostedDatasetIntegrationTest.java

```diff
@@ -311,7 +315,8 @@ private void testSelfHostedDatasetLifecycle(String ingestBucket, boolean dedicat
             .toList();
 
     for (DRSObject drsObject : collect) {
-      TestUtils.validateDrsAccessMethods(drsObject.getAccessMethods(), stewardToken);
+      TestUtils.validateDrsAccessMethods(
+          drsObject.getAccessMethods(), stewardToken, SHOULD_ASSERT_HTTPS_ACCESSIBILITY);
```

Comment:
If this is the only place the check is disabled, it might be easier to catch the thrown assertion and log the exception here, instead of threading through a new argument to the assertion method.

Reply:
That's a good idea.  But here are some other assertions that happen after the failing one though, so we wouldn't get those checked if we allow it to throw.

Reply:
Sorry I wasn't clear, I meant you'd write:
```
      try {
        TestUtils.validateDrsAccessMethods(
            drsObject.getAccessMethods(), stewardToken, SHOULD_ASSERT_HTTPS_ACCESSIBILITY);
      } catch (AssertionError e) {
          logger.warn(...);
      }
```

This way the logging is kept near the code that's forcing the need to log the output, and the boolean parameter doesn't need to be added to the assertion APIs.